src/tasks/misc.ts

```diff
@@ -86,6 +88,14 @@ import { ROUTE_WAIT_TO_NCFORCE } from "../route";
 
 const meatBuffer = 1000;
 
+const sheriffReady = Clan.getWhitelisted().find(
```

Comment:
Could you pull out the two Clan.getWhitelisted() calls into a single variable as well? So we only make one web request.